Documentation/Testing.md

```diff
@@ -0,0 +1,19 @@
+# Getting started with testing
+
+Most tests can be run without any special commands or processes. Running the following command from the project's root directory will restore packages, build the solution, and run the full CoreWCF test suite:
+
+    dotnet test
+
+## Running integration tests
+
+Some integration tests are dependent on external services (e.g. `CoreWCF.RabbitMQ.Tests` require access to a RabbitMQ host), thereby requiring additional setup before those tests can run successfully. The steps below will create local containers hosting all dependent services required by the integration tests.
+
+1. [Install docker v19.03.0+](https://docs.docker.com/engine/install/) to your development environment via Docker Desktop or the docker engine
+2. From the root directory of the CoreWCF repo, run the following command to start the RabbitMQ container:
+
+        docker-compose up -d
+3. Run tests as usual, using your IDE or running `dotnet test` from the command line.
```

Comment:
Can you include the command to stop the containers and clean up in the instructions?

Reply:
Good catch- added instructions